### ISSUE TITLE
Warning message if meyda can't be initialized

### ISSUE THREAD

**Issue by TimPietrusky** [vcync/modV#416]
**Is your feature request related to a problem? Please describe.**
When there is an error with meyda, there is only an error message in the console like `DOMException: Could not start video source`, but modV just keeps working. 

**Describe the solution you'd like**
A warning message as a pop up or something like that to inform the user what is happening and what they should do. Maybe a link to a page that has certain options on how to resolve this. 

**Describe alternatives you've considered**
Open the dev console, but that is not a good thing for endusers.

**Comment by 2xAA** [comment]
@TimPietrusky Do you have a situation where the MediaStream does not initialize? I haven't seen this since modV 1.5.

**Comment by TimPietrusky** [comment]
@2xAA yeah I had this when I was starting modV 2 in dev mode AND modV 3 build version. Will try to reproduce this.

**Comment by 2xAA** [comment]
@TimPietrusky I think this was fixed with https://github.com/vcync/modV/commit/00be42f140dde54c40916c30f1a3583199319e7e - please confirm.
